service-api/module/Application/src/Aws/Secrets/AwsSecretsCache.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Aws\Secrets;
+
+use Application\Aws\Secrets\Exception\InvalidSecretResponseException;
+use Aws\SecretsManager\SecretsManagerClient;
+use Laminas\Cache\Exception\ExceptionInterface;
+use Laminas\Cache\Storage\StorageInterface;
+
+class AwsSecretsCache
+{
+    private const NAMESPACE_AWS = 'aws';
+
+    public function __construct(
+        private readonly string $prefix,
+        private readonly StorageInterface $storage,
+        private readonly SecretsManagerClient $client
+    ) {
+    }
+
+    /**
+     * @throws ExceptionInterface
+     * @throws InvalidSecretResponseException
+     */
+    public function getValue(string $name): string
+    {
+        $name = $this->prefix . '/' . $name;
+        $key = self::NAMESPACE_AWS . ':' . $name;
+
+        /*if ($this->storage->hasItem($key)) {
```

Comment:
Worth just deleting this if you're not using it.

Reply:
But feels like you should use it, because otherwise you always go to the network for secret values.

Reply:
Ah yes, I only commented this out for debugging when keys were not being read properly due to a formatting issue. Will update this now :)